explorer/src/lib/components/list-item/ListItem.css

```diff
@@ -10,6 +9,12 @@
   position: relative;
 }
 
+.details-list__term > div {
```

Comment:
Is the extra element really needed?
If yes: as we agreed to follow BEM, you should give it a class name and not refer to it by its tag name.